### PR TITLE
Add com.chez.GrafX2

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

File: com.chez.GrafX2.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    url: https://gitlab.com/GrafX2/grafX2/-/archive/v2.5/grafX2-v2.5.tar.gz
+    sha256: 5ae1ee5ff34eab3955d140ca02a00808c730cd0848ed39b2f0c5592df0a23684
+  - type: file
+    url: https://sourceforge.net/projects/recoil/files/recoil/4.2.0/recoil-4.2.0.tar.gz
+    sha256: 4e8c8e3048b143654da49cc1187a0b26679c9f97f841edc0ba9e0b2e2e7f18e5
+    dest: 3rdparty/archives # taken from https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/grafx2
+  - type: file
+    path: grafx2.appdata.xml # taken from https://src.fedoraproject.org/rpms/grafx2/tree/master
+  build-commands:
+  - sed -i 's|/usr/local|/app|' ./Makefile
```
Comment (Alexander-Wilms) [inline com.chez.GrafX2.yaml]: make-args somehow didn't work

File: com.chez.GrafX2.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - sed -i 's|/usr/local|/app|' ../3rdparty/recoil-4.2.0/Makefile
+  - mkdir -p /app/share/grafx2/scripts/samples_2.4/picture/others-8bit
```
Comment (Alexander-Wilms) [inline com.chez.GrafX2.yaml]: cp otherwise failed with `no such file or directory`. I'll file a bug report upstream.

**Comment by nedrichards** [comment]
bot, build com.chez.GrafX2

**Comment by Alexander-Wilms** [comment]
Actually added the submodule now

**Comment by nedrichards** [comment]
bot, build com.chez.GrafX2

**Comment by Alexander-Wilms** [comment]
All builds succeeded

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.chez.GrafX2

**Comment by Alexander-Wilms** [comment]
Thanks!

**Pull request closed without merging**